### PR TITLE
Build testimonials page and make minor edits to landing page

### PR DESCRIPTION
**Pull request by marisid**
Related to #24 and #28

### CODE REVIEW

**Comment by marisid** [comment]
@des-des please hold on before merging/reviewing as we need to move the `index.html` file outside the `html` folder so that it is deployable on github pages. I'm doing that now!

**Comment by des-des** [comment]
@marisid re-assign me when ready

**Comment by marisid** [comment]
New commits relate to #37

File: html/exercises.html [hunk 1]
```diff
@@ -57,6 +57,6 @@ <h2 id="beliefs-exercise">Exploring your core beliefs</h2>
         </div>
     <script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js'></script>
     <script src='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js'></script>
-    <script src="../js/app.js"></script>
+    <!-- <script src="../js/app.js"></script> -->
```
Comment (des-des) [inline html/exercises.html]: :fire:

File: js/testimonials.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+(function() {
```
Comment (des-des) [inline js/testimonials.js:1]: Yay

**Review by des-des — CHANGES_REQUESTED** [review]

**Pull request merged**